src/main/java/bit/javaoop/employeesPackage/AbstractEmployee.java

```diff
@@ -1,8 +1,11 @@
-package bit.javaoop;
+package bit.javaoop.employeesPackage;
+
+import bit.javaoop.managersPackage.Manager;
 
 public abstract class AbstractEmployee {
     private String name;
     private double salary;
+    public double satisfyingSalary;
```

Comment:
tą zmienną przenióśłbym już do konkretnych implementacji mam tu na myśli klasę employee bądź BudgetManager itp. i zamieniłbym ją na stałą w ten sposób:
` private static double SATISFYING_SALARY = 10_000`
Zakładamy, że wysokość satysfakcjonującej wypłaty jest zależna od stanowiska:)

Ale generalnie tak jak zrobiłeś to też nie jest jakiś błąd tylko powinna być prywatna mieć getter i ustawiana w konstruktorze w takim wypadku reprezentowało inny design aplikacji taki, że wysokość satysfakcjonującej wypłaty jest zależna od osoby.